README.md

```
@@ -106,6 +106,71 @@ An email or phone number must be sent with the `user` object. This is how Laylo
+      price: 10,
+    },
+    {
+      name: "GA Pass",
+      quantity: 2,
+      price: 20,
+    },
+  ],
+});
+```
+
+**eCommerce**
```

Markdownlint

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)